packages/@webex/plugin-cc/test/unit/spec/services/config/index.ts

```diff
@@ -2,10 +2,21 @@ import {WebexSDK} from '../../../../../src/types';
 import AgentConfigService from '../../../../../src/services/config';
 import HttpRequest from '../../../../../src/services/core/HttpRequest';
 import {WCC_API_GATEWAY} from '../../../../../src/services/constants';
+import {CONFIG_FILE_NAME} from '../../../../../src/constants';
 import MockWebex from '@webex/test-helper-mock-webex';
 import LoggerProxy from '../../../../../src/logger-proxy';
 import * as util from '../../../../../src/services/config/Util';
 
+jest.mock('../../../../../src/logger-proxy', () => ({
+  __esModule: true,
```

Comment:
question. what problem does this solve?

Reply:
We need to do this so as to mock the module Sreekanth, otherwise the tests will not run.